src/main/webapp/app/exercises/programming/manage/instructions-editor/programming-exercise-editable-instruction.component.ts

```diff
@@ -275,10 +273,10 @@ export class ProgrammingExerciseEditableInstructionComponent implements AfterVie
         return this.programmingExerciseParticipationService.getLatestResultWithFeedback(templateParticipationId).pipe(
             rxMap((result) => (!result || !result.feedbacks ? throwError('no result available') : result)),
             rxMap(({ feedbacks }: Result) =>
-                compose(
-                    map(({ text }) => text),
-                    sortBy('text'),
-                )(feedbacks),
+                feedbacks!
+                    .filter((feedback) => feedback.text)
+                    .map((feedback) => feedback.text!)
```

Comment:
Why do you do this change? If in theory there is a feedback without text this would get removed and suddenly the numbers would not fit anymore.

Reply:
Good catch! Removed the additional filter in e466dcf77ffeb7ff45116ffa9f8e5cebb9b5bb2b.